jobrunner/lib/git.py

```diff
@@ -18,6 +18,15 @@
 # See `commit_already_fetched`
 SENTINEL_TAG_PREFIX = "fetched/"
 
+# Prevent git from ever prompting for credentials. Hat tip:
+# https://serverfault.com/a/1054253
```

Comment:
What a wail of frustration!

Reply:
I'm just delighted that it was someone else's wail, from which I get to beneft!